### PR TITLE
Maintenance work to keep the cookbook up-to-date

### PR DESCRIPTION
**Pull request by dpattmann**
Just some maintenance
-   Chef 13 support
-   Rubocop update
-   Kitchen update

### CODE REVIEW

**Comment by dpattmann** [comment]
@nathantsoi any chance to get this merged? :-)

**Comment by nathantsoi** [comment]
awesome @dpattmann, thx man. sry i missed this. yep, ill test and merge

any thoughts @shortdudey123, @ButterflyNetwork or anyone else?

File: .kitchen.yml [hunk 1]
```diff
@@ -3,59 +3,30 @@ driver_plugin: vagrant
...
-- name: fedora-19
-  driver_config:
-    box: opscode_fedora-19_chef-provisionerless
-    box_url: https://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_fedora-19_chef-provisionerless.box
-- name: fedora-20
-  driver_config:
-    box: opscode_fedora-20_chef-provisionerless
-    box_url: https://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_fedora-20_chef-provisionerless.box
+  - name: ubuntu-12.04
+  - name: ubuntu-14.04
```
Comment (shortdudey123) [inline .kitchen.yml:11]: might as well add 16.04 in too

**Comment by shortdudey123** [comment]
looks pretty similar to https://github.com/3ofcoins/chef-cookbook-hostname/pull/59

**Comment by dpattmann** [comment]
@shortdudey123 Oh yes, you're right, haven't seen https://github.com/3ofcoins/chef-cookbook-hostname/pull/59.

Can we merge https://github.com/3ofcoins/chef-cookbook-hostname/pull/59 to this cookbook without breaking anything?

**Comment by shortdudey123** [comment]
The forks look to have diverged enough where it might take a bit to bring in the changes

**Comment by dpattmann** [comment]
@nathantsoi any update? :slightly_smiling_face:

**Review by nathantsoi — APPROVED** [review]

**Comment by nathantsoi** [comment]
thx guys!

**Pull request merged**